Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
I agree, that would be much nicer than two separate lines in the DSL. I'll see about changing this style :smile: